doc/source/whatsnew/v0.23.0

```diff
@@ -362,3 +362,4 @@ Other
 
 - Improved error message when attempting to use a Python keyword as an identifier in a ``numexpr`` backed query (:issue:`18221`)
 - Bug in :class:`Timestamp` where comparison with an array of ``Timestamp`` objects would result in a ``RecursionError`` (:issue:`15183`)
```

Comment:
pls move both this and the one higher to conversion. pls dont' put things in Other that fit elsewhere.

Reply:
OK.  Out of curiosity, what is the heuristic that gets this lumped into "conversion"?